### PR TITLE
update download/flavours to VBT

### LINKED ISSUE #1647: /download/ubuntu-flavours/ to VBT

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* started on converting the download section
 * flavours - http://0.0.0.0:8001/download/ubuntu-flavours
* NOTE: the hover-menu class has made the nav very complicated, I think we need to look at this

## Issue / Card

Fixes #1647 

## Screenshots

[demo](http://www.ubuntu.com-v1-download-flavours.demo.haus/download/ubuntu-flavours)

image

### CODE REVIEW

File: templates/download/_base_download-v1.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{% extends "templates/base-v1.html" %}
+
```
Comment (anthonydillon) [inline templates/download/_base_download-v1.html]: Missing secondary nav? https://github.com/canonical-websites/www.ubuntu.com/blob/master/templates/download/_base_download-v1.html

File: templates/download/ubuntu-flavours.html [hunk 2]
```diff
@@ -7,130 +7,100 @@
 full Ubuntu archive for packages and updates.{% endblock %}
 
 {% block second_level_nav_items %}
-    <div class="strip-inner-wrapper">
-        {% include "templates/_nav_breadcrumb.html" with section_title="Downloads" subsection_title="Ubuntu flavours" page_title="Ubuntu flavours"  %}
-    </div><!-- /.strip-inner-wrapper -->
+{% include "templates/_nav_breadcrumb-v1.html" with section_title="Downloads" subsection_title="Ubuntu flavours" page_title="Ubuntu flavours"  %}
```
Comment (anthonydillon) [inline templates/download/ubuntu-flavours.html]: Under tabbed.

File: templates/download/_nav_secondary-v1.html [hunk 3]
```diff
@@ -1,9 +1,8 @@
...
-    <li><a{% if level_2 == 'cloud' %} class="is-active"{% endif %} href="/download/cloud">Cloud</a></li>
-    <li><a{% if level_2 == 'server' %} class="is-active"{% endif %} href="/download/server">Server</a></li>
-    <li><a{% if level_2 == 'desktop' %} class="is-active"{% endif %} href="/download/desktop">Desktop</a></li>
-    <li><a{% if level_2 == 'ubuntu-kylin' or level_2 == 'ubuntu-kylin-zh-CN' %} class="is-active"{% endif %} href="/download/ubuntu-kylin">Ubuntu Kylin</a></li>
-    <li><a{% if level_2 == 'alternative-downloads' %} class="is-active"{% endif %} href="/download/alternative-downloads">Alternative downloads</a></li>
-    <li><a{% if level_2 == 'ubuntu-flavours' %} class="is-active"{% endif %} href="/download/ubuntu-flavours">Ubuntu flavours</a></li>
-  </ul>
+<ul class="{% if list_class %}{{ list_class }}{% else %}nav-secondary__menu p-inline-list{% endif %}">
+  <li{% if not list_class %} class="p-inline-list__item"{% endif %}><a class="{% if not list_class %}p-inline-list__link {% endif %}{% if level_1 == 'download' and not level_2 %} is-active"{% endif %}" href="/download">Overview</a></li>
```
Comment (barrymcgee) [inline templates/download/_nav_secondary-v1.html]: Is `{% if not list_class %}` required? I didn't add these on the other section secondary navs?

Comment (pmahnke) [inline templates/download/_nav_secondary-v1.html]: you don't have dropdowns for /about or /legal in the primary orange nave

**Pull request merged**